src/components/BadgesFromPkg.tsx

```diff
@@ -40,6 +42,7 @@ export const defaultBadges: ReadonlyArray<BadgeName> = [
   "devDependenciesBadge",
   "semanticRelease",
   "jsxReadme",
+  "npmBundleSize",
```

Comment:
My bad - you will also have to move it in this list as this is used for sorting. Maybe you could also add a comment that this array's order determines the order in which components are displayed.

Reply:
Sorry, I didn't realize that. I've changed the code per your suggestions.